.github/workflows/cpp.yml

```diff
@@ -197,12 +197,21 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        os:
-          - windows-2019
         include:
           - os: windows-2019
-            name: Windows 2019
+            name: Windows 2019 x64
             generator: Visual Studio 16 2019
+            platform: x64
+            openssl-n-bits: 64
```

Comment:
Hmm, what is this for? Can I find some documentation for this somewhere? A Google search yields nothing.

Reply:
These are build variables in the GitHub workflow.
Platform is used to specify the CPU configuration to build using MSBuild.
openssl-n-bits is used to expand the directory name that OpenSSL got installed to using the chocalatey package manager.

Reply:
I understand about `platform`, but I can't find any reference to `openssl-n-bits` anywhere. Did I miss something?

Reply:
but `${{ matrix.openssl-n-bits }}` is never used below, so how does it work?